project/OcsBundle.scala

```diff
@@ -425,7 +425,7 @@ trait OcsBundle {
   lazy val bundle_edu_gemini_too_event = 
     project.in(file("bundle/edu.gemini.too.event")).dependsOn(
       bundle_edu_gemini_shared_skyobject,
-      bundle_edu_gemini_pot,
+      bundle_edu_gemini_pot % "test->test;compile->compile",
```

Comment:
I wanted to use a class in the test code of the `pot` bundle.  This bizarre looking appendage makes the `pot` test classes available to the tests in the `too_event` bundle.  I'm sure there's a better, more complete, explanation but having obtained the result I wanted I didn't look deeper.